integration-cli/docker_cli_create_test.go

```diff
@@ -60,6 +60,27 @@ func (s *DockerSuite) TestCreateArgs(c *check.C) {
 
 }
 
+// Make sure we can grow the container's rootfs at creation time.
+func (s *DockerSuite) TestCreateGrowRootfs(c *check.C) {
+	testRequires(c, Devicemapper)
```

Comment:
@mYmNeo here you go : https://github.com/docker/docker/blob/666563b190b7c97dc9db334ef03768ee8432b20e/integration-cli/requirements.go#L121